Sorry, I left the build running and forgot to check the result. Here is the full error:

```
pizauth> stripping (with command strip and flags -S) in  /nix/store/584flvjgym1gcy5xiqzplbh82g1jwb1f-pizauth-1.0.5/bin
pizauth-x86_64-darwin> error: linking with `/nix/store/d2jm2gdf5qmyfbqk579f7h65sbi497sr-clang-wrapper-16.0.6/bin/cc` failed: exit status: 1
pizauth-x86_64-darwin>   |
pizauth-x86_64-darwin>   = note: env -u IPHONEOS_DEPLOYMENT_TARGET -u TVOS_DEPLOYMENT_TARGET -u XROS_DEPLOYMENT_TARGET LC_ALL="C" PATH="/nix/store/3dbzxw08y3ryhqmpgfyks6syc422yy84-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/bin:/nix/store/3dbzxw08y3ryhqmpgfyks6syc422yy84-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/bin:/nix/store/3dbzxw08y3ryhqmpgfyks6syc422yy84-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/bin:/nix/store/ihphbwmww3g33pjcm3b28zsbp0mvp9rf-cargo-1.82.0/bin:/nix/store/cpm6ml11szp69905j98md1a8d4f3sk7w-cargo-auditable-0.6.2/bin:/nix/store/jzd1zpz3fmr67i0my6jnj9azpz1fyvz1-auditable-cargo-1.82.0/bin:/nix/store/ihphbwmww3g33pjcm3b28zsbp0mvp9rf-cargo-1.82.0/bin:/nix/store/r1c71slidpiav3bhc4n2bqk24pjm0j3g-rustc-wrapper-1.82.0/bin:/nix/store/d2jm2gdf5qmyfbqk579f7h65sbi497sr-clang-wrapper-16.0.6/bin:/nix/store/lmpv596dnw3x2la3m4gfsq6yw7d5ikyc-clang-16.0.6/bin:/nix/store/0smc7dkkl344m8wwfmhb1icgfbfx2j4x-coreutils-9.5/bin:/nix/store/fb5iqyd96rvix58v5mblc05143l46mlj-cctools-binutils-darwin-wrapper-1010.6/bin:/nix/store/b8hk1kc983p6x1lvghmalgv46mdp8f8d-cctools-binutils-darwin-1010.6/bin:/nix/store/93pk7jcvsisharnzf2d1qrnd2d1llw0a-xcbuild-0.1.1-unstable-2019-11-20-xcrun/bin:/nix/store/0smc7dkkl344m8wwfmhb1icgfbfx2j4x-coreutils-9.5/bin:/nix/store/abgsnf0ddiqawx2q72aq1s96v179lv9p-findutils-4.10.0/bin:/nix/store/kh2zly4fclxi8w99wfi7znzpw4lg20x2-diffutils-3.10/bin:/nix/store/61xrjs2m9c3vbcl1md124l58y9y7pnxj-gnused-4.9/bin:/nix/store/1l3lp7kpsnhkypsgkb4yk5jxa8wvwdan-gnugrep-3.11/bin:/nix/store/q39p4s4bpkr8vai4gcdmwvvmwmaszb4f-gawk-5.3.1/bin:/nix/store/zpb3vsqvmpyf377c2gxcwqv1xsjyndh1-gnutar-1.35/bin:/nix/store/rxk1c2a7qa8pd3w4kcyckwgscq9y872f-gzip-1.13/bin:/nix/store/w4cyi5zyxx7j4qv0x5h0rrpkmjmx6bg7-bzip2-1.0.8-bin/bin:/nix/store/bsgyj981nm8qrj467w1xzp52c4289nca-gnumake-4.4.1/bin:/nix/store/qwg7nqq0y2lzc94pdaa00li2z12h4jl4-bash-5.2p37/bin:/nix/store/i5imv15kmp7lwcx15vzkbpjxpd40bzpf-patch-2.7.6/bin:/nix/store/1zz119aarsbn8xk5l7dqqidprh5riny9-xz-5.6.3-bin/bin:/nix/store/fhd3pi9rw6vpqryim5n1ifkgpsa80yhl-file-5.45/bin" VSLANG="1033" ZERO_AR_DATE="1" "/nix/store/d2jm2gdf5qmyfbqk579f7h65sbi497sr-clang-wrapper-16.0.6/bin/cc" "-arch" "x86_64" "-m64" "/private/tmp/nix-build-pizauth-1.0.5.drv-0/rustcMbds8Y/symbols.o" "/private/tmp/nix-build-pizauth-1.0.5.drv-0/source/target/x86_64-apple-darwin/release/deps/pizauth-ab65332b678bc469.pizauth.c67da596f9a98105-cgu.0.rcgu.o" "/private/tmp/nix-build-pizauth-1.0.5.drv-0/rustcMbds8Y/libring-ec456d6aee313da1.rlib" "/nix/store/3dbzxw08y3ryhqmpgfyks6syc422yy84-rustc-1.82.0/lib/rustlib/x86_64-apple-darwin/lib/libcompiler_builtins-df3bf8574a3856a3.rlib" "-framework" "CoreFoundation" "-framework" "SystemConfiguration" "-framework" "CoreFoundation" "-liconv" "-lSystem" "-lc" "-lm" "-L" "/private/tmp/nix-build-pizauth-1.0.5.drv-0/source/target/x86_64-apple-darwin/release/build/ring-89122a11e6ad9b60/out" "-o" "/private/tmp/nix-build-pizauth-1.0.5.drv-0/source/target/x86_64-apple-darwin/release/deps/pizauth-ab65332b678bc469" "-Wl,-dead_strip" "-nodefaultlibs" "/private/tmp/nix-build-pizauth-1.0.5.drv-0/source/target/x86_64-apple-darwin/release/deps/pizauth_audit_data.o" "-Wl,-u,_AUDITABLE_VERSION_INFO"
pizauth-x86_64-darwin>   = note: Undefined symbols for architecture x86_64:
pizauth-x86_64-darwin>             "_utimensat", referenced from:
pizauth-x86_64-darwin>                 nix::with_nix_path_allocating::heb3547e925ac2b08 in pizauth-ab65332b678bc469.pizauth.c67da596f9a98105-cgu.0.rcgu.o
pizauth-x86_64-darwin>                 std::sys::backtrace::__rust_begin_short_backtrace::h28ef0f3d0ce3304f in pizauth-ab65332b678bc469.pizauth.c67da596f9a98105-cgu.0.rcgu.o
pizauth-x86_64-darwin>           ld: symbol(s) not found for architecture x86_64
pizauth-x86_64-darwin>           clang-16: error: linker command failed with exit code 1 (use -v to see invocation)
pizauth-x86_64-darwin>
pizauth-x86_64-darwin>
pizauth-x86_64-darwin> error: could not compile `pizauth` (bin "pizauth") due to 1 previous error
```

Can confirm that adding `apple-sdk_11` to `buildInputs` fixes the build on `x86_64-darwin`.

--- a/pkgs/by-name/pi/pizauth/package.nix
+++ b/pkgs/by-name/pi/pizauth/package.nix
@@ -3,8 +3,8 @@
   rustPlatform,
   fetchFromGitHub,
   installShellFiles,
+  apple-sdk_11,
   stdenv,
-  darwin,
 }:
 
 rustPlatform.buildRustPackage rec {
@@ -14,15 +14,16 @@ rustPlatform.buildRustPackage rec {
   src = fetchFromGitHub {
     owner = "ltratt";
     repo = "pizauth";
-    rev = "pizauth-${version}";
+    rev = "refs/tags/pizauth-${version}";
     hash = "sha256-9NezG644oCLTWHTdUaUpJbuwkJu3at/IGNH3FSxl/DI=";
   };
 
   cargoHash = "sha256-Lp5ovkQKShgT7EFvQ+5KE3eQWJEQAL68Bk1d+wUo+bc=";
 
-  nativeBuildInputs = [ installShellFiles ];
+  # pizauth cannot be built with default apple-sdk_10 on x86_64-darwin, pin to 11
+  buildInputs = lib.optional stdenv.hostPlatform.isDarwin apple-sdk_11;
 
-  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [ darwin.apple_sdk.frameworks.Security ];
+  nativeBuildInputs = [ installShellFiles ];
 
   postInstall = ''
     installShellCompletion --cmd pizauth \

